### DISCUSSION TITLE
Chaos Engineering Resources

### DISCUSSION THREAD

**Discussion by prithvi1307** [litmuschaos/litmus#2854] [General]
Let's post Chaos Engineering resources out here for community members

**Comment by neelanjan00** [comment]
**A Beginner's Practical Guide to Containerization and Chaos Engineering with Litmus Chaos 2.0** 
This two blog series is a beginner-focused guide on getting started with containerization from the ground up, starting from containers all the way to chaos engineering. 

The first blog is all about the world of microservices and containerization, where you'd be gently covering all that you need to know about Docker, Kubernetes. It also has a thorough step-by-step walkthrough of deploying a dead simple web app using Docker and Kubernetes using Minikube while covering every essential detail in between. [Read Now](https://dev.to/neelanjan00/part-1-a-beginner-s-practical-guide-to-containerisation-and-chaos-engineering-with-litmuschaos-2-0-3h5c)

The second blog is all about the world of chaos engineering. It discusses the motivations, principles, and practicality of chaos engineering, as well as the principles of cloud-native chaos engineering, and finally about LitmusChaos. It also includes a step-by-step tutorial to induce `pod-delete` chaos into the application that had been deployed in the first part of the blog, using the LitmusChaos 2.0 portal. [Read Now](https://dev.to/neelanjan00/part-2-a-beginner-s-practical-guide-to-containerisation-and-chaos-engineering-with-litmuschaos-2-0-253i)

**Comment by neelanjan00** [comment]
**Getting Started with Litmus 2.0 in Google Kubernetes Engine**
This blog is a beginner friendly step-by-step guide on how to setup LitmusChaos 2.0 in a GKE cluster. [Read Now](https://dev.to/neelanjan00/getting-started-with-litmus-2-0-in-google-kubernetes-engine-4obf)
